iina/ThumbnailCache.swift

```diff
@@ -50,7 +50,7 @@ class ThumbnailCache {
         return false
       }
 
-      let cacheVersion = Int(file.read(type: UInt8.self))
+      let cacheVersion = file.read(type: UInt8.self)
```

Comment:
Yeah also agree, though taking it a step further, I think it could be a good idea to use typealiases for the three metadata items to make it clearer instead of keeping track of different int types. Let me know thoughts on the new commit.

Reply:
Good idea.